api/src/main/java/org/eclipse/microprofile/concurrent/ThreadContextConfig.java

```diff
@@ -165,4 +167,40 @@
      * includes one or more of the same types as this set.</p>
      */
     String[] unchanged() default {};
+    
+        /**
+        * Util class used for inline creation of {@link ThreadContextConfig} annotation instances.
+        */
+        public final class ThreadContextConfigLiteral extends AnnotationLiteral<ThreadContextConfig> implements ThreadContextConfig {
+
+            public static final ThreadContextConfigLiteral DEFAULT_INSTANCE = 
+                of(new String[]{ThreadContext.TRANSACTION}, new String[]{ThreadContext.ALL_REMAINING}, new String[]{ThreadContext.ALL_REMAINING});
```

Comment:
There is a typo here specifying ALL_REMAINING for both the propagated value as well as unchanged.
Should be:
`of(new String[]{ThreadContext.TRANSACTION}, new String[]{}, new String[]{ThreadContext.ALL_REMAINING});`

Reply:
Good catch, thanks!